### PR TITLE
do not ignore CVE vulns in base images

### PR DESCRIPTION
**Pull request by jacob-delgado**

### CODE REVIEW

**Comment by jacob-delgado** [comment]
For https://github.com/istio/istio/issues/30601

File: release/build.sh [hunk 1]
```diff
@@ -46,7 +46,6 @@ MANIFEST=$(cat <<EOF
 version: ${VERSION}
 docker: ${DOCKER_HUB}
 directory: ${WORK_DIR}
-ignoreVulnerability: true
```
Comment (shamsher31) [inline release/build.sh:49]: can we just set it to false?

Comment (jacob-delgado) [reply release/build.sh:49]: This is the way it was set originally https://github.com/istio/release-builder/pull/531/files

But I can change it

Comment (shamsher31) [reply release/build.sh:49]: Thanks

**Review by shamsher31 — APPROVED** [review]

**Review by stevenctl — APPROVED** [review]

File: release/build.sh (hunk 1 above)
Comment (ericvn) [reply release/build.sh:49]: I'd have to go back and look, but I think the same script is run for all the commits in master (again, need to look), so in master they would ignore vulnerabilities, Maybe something to add to the release-branch scripts/list if that is the case.

**Pull request merged**